go.mod

```diff
@@ -2,13 +2,19 @@ module github.com/testifysec/witness
 
 go 1.18
 
+replace github.com/testifysec/go-witness => ../go-witness
+
+replace github.com/testifysec/witness => ./witness
```

Comment:
Check out go workspaces for local development of multiple modules simultaneously.